src/Administration/Resources/app/administration/src/module/sw-mail-template/page/sw-mail-template-index/sw-mail-template-index.html.twig

```diff
@@ -81,16 +81,56 @@
     {% block sw_mail_template_list_content %}
     <template #content>
         <sw-card-view>
+            {# @deprecated tag:v6.8.0 - The template wrapper will be removed. #}
+            <template v-if="useTabs">
+                {% block sw_mail_template_list_tabs %}
+                <sw-tabs
+                    class="sw-mail-template-list__tabs"
+                    position-identifier="sw-mail-template-index"
```

Comment:
TODO: Run `composer admin:generate-position-identifier-list`, waiting for:
- https://github.com/shopware/shopware/pull/14142